test.js

```diff
@@ -681,7 +681,8 @@ test('uses the custom errorMessage callback if passed in as an option', function
   })
 
   dest.on('data', function (line) {
-    t.equal(line.msg, customErrorMessage)
+    console.log(line.msg)
```

Comment:
remove this

Reply:
whoops, my bad, was just testing something when doing this. thanks, will remove